core/src/ast_builder/expr/function.rs

```diff
@@ -956,15 +956,16 @@ pub fn values<'a, T: Into<ExprNode<'a>>>(expr: T) -> ExprNode<'a> {
 mod tests {
     use crate::{
         ast::DateTimeField,
-        ast_builder::{
-            abs, acos, ascii, asin, atan, calc_distance, cast, ceil, chr, coalesce, col, concat,
-            concat_ws, cos, date, degrees, divide, entries, exp, expr, extract, find_idx, floor,
-            fn_values, format, gcd, generate_uuid, get_x, get_y, greatest, ifnull, initcap,
-            is_empty, last_day, lcm, left, length, ln, log, log10, log2, lower, lpad, ltrim, md5,
-            modulo, now, null, num, pi, point, position, power, radians, rand, repeat, replace,
-            reverse, right, round, rpad, rtrim, sign, sin, skip, sqrt, substr, take, tan,
-            test_expr, text, time, timestamp, to_date, to_time, to_timestamp, upper,
+        ast_builder::function::{
+            abs, acos, ascii, asin, atan, calc_distance, cast, ceil, chr, coalesce, concat,
+            concat_ws, cos, degrees, divide, entries, exp, extract, find_idx, floor, format, gcd,
+            generate_uuid, get_x, get_y, greatest, ifnull, initcap, is_empty, last_day, lcm, left,
+            length, ln, log, log10, log2, lower, lpad, ltrim, md5, modulo, now, pi, point,
+            position, power, radians, rand, repeat, replace, reverse, right, round, rpad, rtrim,
+            sign, sin, skip, sqrt, substr, take, tan, to_date, to_time, to_timestamp, upper,
+            values,
```

Comment:
Is it possible to change the one used here? 
You can also alias it with `*` or `as f`.

Reply:
Thanks for the correction!